src/components/layouts/AdminLayout/AdminLayout.tsx

```diff
@@ -15,6 +19,7 @@ export const AdminLayout: React.FC<AdminLayoutPropsType> = ({ children }) => {
       document.documentElement.classList.add(htmlClass)
     );
     document.body.classList.add("h-full");
+
```

Comment:
Hence the [code smell] comment. I know it doesn't smell right, but can't quite figure out if there is something wrong and how it can be improved. 

Let's leave it as is and check with Design team if we really need to touch elements outside the react root